src/containers/CheckInOut/index.tsx

```diff
@@ -56,13 +57,20 @@ interface CheckInOut {
   checkoutTime: Date;
   timeId: string;
   employee: Employees;
-  currenDate: Date;
+  currentDate: Date;
 }
 
 const CheckInOut: React.FC = () => {
+  const { currentUserData } = useCurrentUser();
+  const role = currentUserData?.data?.data?.role;
+
+  let isEmployee = false;
+  if (role === 'Employee') {
+    isEmployee = true;
+  }
```

Comment:
same here instead of these 4 line we can use
`const [isEmployee] = useState(role === 'Employee')`